kano/network.py

```diff
@@ -535,6 +546,75 @@ def reload_kernel_module(device_vendor='148f', device_product='5370', module='rt
     return reloaded
 
 
+def connection_result(iface, wpa_file, connection_timeout, verbose=False):
```

Comment:
Maybe change the function name to reflect that it is also doing the connect, currently it sounds like it is just figuring out whether you are connected after you've already attempted to connect. Maybe something like `attempt_connection()`,  `try_connect()` or even `do_connect()`.

Reply:
Totally agreed, fixed.